keyboards/dumbo/readme.md

```diff
@@ -0,0 +1,17 @@
+# Dumbo
+
+![Dumbo](https://i.imgur.com/EEqCZsgl.jpg)
+
+Inspired by the corne, kyria and chimera ergo 42, the dumbo is a split keyboard ready for a comfortable future (read: bluetooth and ergo).
+
+The Dumbo is a small split ergonomic keyboard with 44 keys (outer columns can be broken of to make it 38 keys), heavy columnar stagger, small and comfortable thumb cluster, support for two rotary encoders per half, support for oleds, and optional support for a jst-ph two pin connector to attach a battery and an on/of switch to power bluetooth pro micro replacements like the nice!nano.
```

Comment:
@qmk/collaborators 

![image](https://user-images.githubusercontent.com/18669334/89265126-e0457a00-d5e8-11ea-97e8-1c0abd1b1a25.png)

`LAYOUT_split_3x6_4`?

Reply:
yes, that is the correct layout (if that is what you mean).

Reply:
@adamnaldal We have a feature where keyboards that share a similar physical layout can use a special keymaps directory where users can save one keymap that can be applied to any keyboard that supports the layout.

- [Community Layouts - QMK Firmware Docs](https://docs.qmk.fm/#/feature_layouts)

In this case, your layout is a split keyboard with main keywells of 3 rows by 6 columns, with four thumb-oriented keys (technically 3 + clickable encoder) on each half. I'm asking if we should set your keyboard to support said feature, which requires some minor changes.

Reply:
Note that the `split_3x6_4` layout does not actually exist yet, but it would be a one liner in rules.mk to add support for this board if it did in the future, if the LAYOUT macro were renamed now.

Reply:
Oh that's super cool i will absolutely add the suggested changes to support this in the future 👍